app/javascript/components/settings-company-tags-entry-form/index.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import MiqFormRenderer from '@@ddf';
+import PropTypes from 'prop-types';
+import createSchema from './schema';
+import { http } from '../../http_api';
+
+/** component to render the Category Entries Add/Edit Form */
+const SettingsCompanyTagsEntryForm = ({ categoryId, entryId, callbackAction }) => {
+  const ENTRY_ACTIONS = {
+    ADD: 'add',
+    SAVE: 'save',
+    CANCEL: 'cancel',
+  };
+  const newRecord = entryId === 'new';
+  const [{ initialValues, isLoading }, setState] = useState({
+    isLoading: !!categoryId,
+  });
+
+  useEffect(() => {
+    if (newRecord) {
+      setState({ isLoading: false });
+    } else {
+      http.get(`/ops/category_entries/${categoryId}?entry_id=${entryId}`).then((initialValues) => {
+        setState({ initialValues, isLoading: false });
+      });
+    }
+  }, [categoryId]);
+
+  const onSubmit = (values) => {
+    const params = newRecord
+      ? { data: { ...values, button: ENTRY_ACTIONS.ADD }, url: `/ops/ce_accept/${categoryId}` }
+      : { data: { ...values, button: ENTRY_ACTIONS.SAVE }, url: `/ops/ce_accept/${categoryId}?entry_id=${entryId}` };
+
+    http.post(params.url, params.data, { skipErrors: [400, 500] })
+      .then((response) => {
+        if (response.type === 'success') {
+          const action = (newRecord) ? __('created') : __('saved');
+          add_flash(sprintf(__('Category entry "%s" was %s successfully'), values.name, action), response.type);
```

Comment:
Building a string like this could cause problems for the translation team, there's loss of context by pulling the verb out of the sentence.

Reply:
```
const message = (newRecord) ? sprintf(__('Category entry "%s" was created successfully'), values.name) : sprintf(__('Category entry "%s" was saved successfully'), values.name);
add_flash(message, response.type);
```